yt_dlp/utils/_utils.py

```diff
@@ -1887,7 +1889,7 @@ def setproctitle(title):
     buf = ctypes.create_string_buffer(len(title_bytes))
     buf.value = title_bytes
     try:
-        libc.prctl(15, buf, 0, 0, 0)
+        libc.prctl(PR_SET_NAME, buf, 0, 0, 0)
```

Comment:
I think we can just add the comment here instead
```suggestion
        # PR_SET_NAME flag from "/usr/include/linux/prctl.h"
        libc.prctl(15, buf, 0, 0, 0)
```